examples/pxScene2d/src/rcvrcore/XModule.js

```diff
@@ -16,10 +16,7 @@ limitations under the License.
 
 */
 
-"use strict";
```

Comment:
Out of curiosity, what is causing the need to remove "use strict"?

Reply:
I thought it is strict by default in node. However after experimenting I see it is not, brought this back